flytekit/configuration/__init__.py

```diff
@@ -459,6 +459,7 @@ class Config(object):
     secrets: SecretsConfig = SecretsConfig()
     stats: StatsConfig = StatsConfig()
     data_config: DataConfig = DataConfig()
+    image_config: ImageConfig = ImageConfig()
```

Comment:
I think this was intentionally left out of here, and put into just the `SerializationSettings` object.  We'll have to add it to FlyteRemote separately.